tests/test_memvid_utils.py

```
@@ -0,0 +1,233 @@
+
+        # Should have multiple chunks
+        assert len(chunks) > 1
+
+        # When chunks are joined (removing overlap), should contain original content
+        unique_content = ""
+        for i, chunk in enumerate(chunks):
+            if i == 0:
+                unique_content += chunk
+            else:
+                # Remove overlap
+                overlap_start = max(0, len(chunk) - (50 - 5))
```

Ruff (0.14.8)

Local variable `overlap_start` is assigned to but never used

Remove assignment to unused variable `overlap_start`

(F841)